plugin/mutate/source/dextool/plugin/mutate/backend/database/standalone.d

```diff
@@ -401,7 +401,7 @@ struct Database {
     }
     // Returns: the status of the mutant
     Nullable!Mutation.Status getMutationStatus(const MutationId id) @trusted {
-        auto stmt = db.prepare(format("SELECT status FROM %s WHERE id IN (SELECT st_id FROM %s WHERE id=:mut_id)",
+        auto stmt = db.prepare(format!"SELECT status FROM %s WHERE id IN (SELECT st_id FROM %s WHERE id=:mut_id)"(
```

Comment:
i recommend moving the string to a line before this and assigned to an immutable variable.